ir/pass_utils.cpp

```diff
@@ -0,0 +1,64 @@
+#include "ir/pass_utils.h"
+
+#include <initializer_list>
+#include <tuple>
+
+#include "lib/log.h"
+
+/// @file
+/// @authors Vladimír Štill
```

Comment:
Nit: A little non-standard, should we use a copyright notice instead?

Reply:
Is there a "standard" copyright notice in modern "P4C"? I'm mostly finding old barefoot notices...

Or I can just skip this.

Reply:
I do not think there is a standard, but the Apache template we use is the most common one. Recent example here: https://github.com/p4lang/p4c/blob/main/midend/def_use.h#L2 

But we also have https://github.com/p4lang/p4c/blob/e30416337cc0e3bf046909324aa7ae0b160f5224/lib/nethash.cpp#L23 here now, so I think we can go with either version.